src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1>Leaderboard</h1>
+    <div id="main-container">
+        <section id="recent-scores-main-container">
+            <div id="recent-scores-top">
+                <h3>Recent scores</h3>
+                <span id="refresh-item" class="clickable">Refresh</span>
+            </div>
+            <ul id="scores-list">
+                <li class="list-element">Name: 100</li>
+                <li class="list-element">Name: 50</li>
+                <li class="list-element">Name: 60</li>
+                <li class="list-element">Name: 70</li>
+                <li class="list-element">Name: 30</li>
+                <li class="list-element">Name: 10</li>
+            </ul>
+        </section>
+        <section id="add-score-container">
+            <h3>Add your score</h3>
+            <form id="new-score-form">
+                <input type="text" placeholder="Your name">
```

Comment:
- [ ] index.html main (which is supposed to be the main wrapper element inside the body) contains only the root element as was mentioned by the previous reviewer. Instead of generating the sections directly inside the HTML file,  I think you should generate the DOM elements from js files. Kindly make this change before you make another review request. :pray:

Reply:
Thanks for the feedback.

I was checking the project specifications and this requirement is not in the project specification.
Or maybe I am not understanding your request. Could you please clarify? 

Also, I was checking some other approved projects to compare and, they don't generate the DOM elements (`<li>`, `<div>`, `<section>`, `<ul>`, etc.) from the js file. 
As the requirement says "**Write plain HTML** markup with minimum styling (just to make the layout), because ..."